### PR TITLE
dev-python/munkres: 1.1.2 version bump

### PR DESCRIPTION
**Pull request by jdowner**

### CODE REVIEW

File: dev-python/munkres/munkres-1.1.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_5,3_6} )
```
Comment (juippis) [inline dev-python/munkres/munkres-1.1.2.ebuild]: Any chance you could test if python3.7 works too?

File: dev-python/munkres/munkres-1.1.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+SRC_URI="https://github.com/bmc/munkres/archive/release-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/${PN}-release-${PV}"
+
+python_test() {
+	"${PYTHON}" ${PN}.py || die
```
Comment (juippis) [inline dev-python/munkres/munkres-1.1.2.ebuild]: `EPYTHON` most likely.

Comment (jdowner) [reply dev-python/munkres/munkres-1.1.2.ebuild]: Fixed.

File: dev-python/munkres/munkres-1.1.2.ebuild (hunk 1 above)
Comment (jdowner) [reply dev-python/munkres/munkres-1.1.2.ebuild]: Done. All of the tests pass for each version.

**Comment by juippis** [comment]
Thanks!

**Pull request closed without merging**